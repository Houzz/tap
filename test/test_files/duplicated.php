<?php
class DuplicateClass {
    public function alpha() {
        return 42;
    }
}
class DuplicateClass {
    public function beta() {
        return 42;
    }
}

function func() {}
function func() {}

