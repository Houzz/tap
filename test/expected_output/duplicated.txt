7 E_DUPLICATED_CLASS
14 E_DUPLICATED_FUNCTION
